src/chat/tools/getConnections.ts

```diff
@@ -133,6 +139,9 @@ export class GetConnectionsTool extends BaseLanguageModelTool<IGetConnectionsPar
       },
       token,
     );
+    stream.progress(
+      `Found ${result.content.length} connections for connectionType: ${parameters.connectionType}.`,
+    );
```

Comment:
Let's add another `if (!parameters.connectionType)` check here in the event the model doesn't pass one, so we can keep the progress message generic instead of showing `...for connectionType: undefined`